### DISCUSSION TITLE
Filter out people starring own repositories

### DISCUSSION THREAD

**Discussion by AykutSarac** [community/community#129095] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

On my feed I'd like to see people I'm following starring repositories but I don't want to see my own repositories getting starred by people.

**Comment by Rishi-Sudhakar** [comment, marked as answer]
I don't think it's possible. However you can, Go to the repository page, click the "Watch" button, and select "Ignoring" to stop receiving notifications about stars and other activities on your own repositories.( for your popular repos, but well it stops all notifications on all activities ^.^  )

**Reply by AykutSarac** [reply]
Thanks for the suggestion Rishi I'll consider this, I think it'd be nice to see this filter option added to the feed.
